pkg/prober/prober.go

```diff
@@ -0,0 +1,158 @@
+package prober
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/improbable-eng/thanos/pkg/component"
+	"github.com/prometheus/common/route"
+)
+
+const (
+	healthyEndpointPath  = "/-/healthy"
+	readyEndpointPath    = "/-/ready"
+	okProbeText          = "thanos %v is %v"
```

Comment:
```suggestion
	okProbeFmt          = "thanos %v is %v"
```
To show it is templated

Reply:
and below (:

Reply:
all should be renamed